packages/mermaid/src/diagrams/gantt/parser/gantt.jison

```diff
@@ -130,6 +131,7 @@ statement
   | excludes {yy.setExcludes($1.substr(9));$$=$1.substr(9);}
   | includes {yy.setIncludes($1.substr(9));$$=$1.substr(9);}
   | todayMarker {yy.setTodayMarker($1.substr(12));$$=$1.substr(12);}
+  | weekday { yy.setWeekday($1.substr(8));$$=$1.substr(8);}
```

Comment:
It would be better to restrict possible values, I'm it would be will take more effort but to return a good error message when it happen

Reply:
Where is the best place to do this validation? In `gantt.jison` or when `setWeekday` is called? I can imagine what this looks like in Javascript, but I have no clue how I would approach this in jison...

Reply:
In `gantt.jison`. Please take a look at `flowchart` or `classDiagram` directions, it would help to see how to restrict possible values in the grammar itself.

Reply:
I've based a solution on the `direction` statement in `flow.jison`. Tests seem okay with it, so I guess I am as well.